composer.json

```diff
@@ -9,13 +9,15 @@
         "php": ">=5.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.3.5"
+        "phpunit/phpunit": "4.3.5",
+        "malkusch/php-mock": "dev-php-5.3"
     },
     "autoload": {
         "psr-4": {
             "Mage\\": "./Mage",
             "Task\\": [".mage/tasks", "../../../.mage/tasks"],
-            "Command\\": [".mage/tasks", "../../../.mage/commands"]
+            "Command\\": [".mage/tasks", "../../../.mage/commands"],
+            "MageTest\\": "./tests/MageTest"
```

Comment:
For unit test classes, please use autoload-dev.
https://getcomposer.org/doc/04-schema.md#autoload-dev